castle/validators/present.py

```diff
@@ -0,0 +1,10 @@
+from castle.exceptions import InvalidParametersError
+
+
+class ValidatorsPresent(object):
+
+    @staticmethod
+    def call(options, *args):
```

Comment:
FWIW, I find this pattern of static `call` method on a class confusing. It seems to me that all the class is doing is namespacing the function... which I don't see any benefit to. It feels like this would be simpler as:
```python
def validate_present(options, *keys):
```

Reply:
we plan to have similar interface in all the sdks